test/TestUtilities/Test.Utility/TestServer/PortReserver.cs

```diff
@@ -1,6 +1,7 @@
-﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved. See License.txt in the project root for license information.
+// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved. See License.txt in the project root for license information.
```

Comment:
We should update the header, this one is out of date.

Reply:
Thanks! Updated with the latest header.